Phonebook.Frontend/src/environments/EnvironmentInterfaces.ts

```diff
@@ -41,3 +50,20 @@ export interface RuntimeEnvironmentInterface {
    **/
   roomPlanningToolUrl: string | null;
 }
+
+export enum Environment {
```

Comment:
It would also be possible to define Environment as `export type Environment = 'development' | 'preview' | 'production'`. It has some benefits compared to an enum but also some drawbacks

Reply:
I will leave it as it is as it currently has better type safety in HTML templates.